Assets/Scripts/DataController.cs

```diff
@@ -138,7 +145,77 @@ void Start()
                 }
             }
         }
+
+        if (markerPrefab != null)
+        {
+            AddMarker("NCP", 0f, 90f, localSiderialTime, colorOrange);
+            AddMarker("SCP", 0f, -90f, localSiderialTime, colorOrange);
+            AddMarker("VE", 0f, 0f, localSiderialTime, colorGreen);
+            AddCircumferenceMarker("equator", colorBlue, markerLineWidth);
+            AddLineMarker("poleLine", colorOrange, GameObject.Find("NCP"), GameObject.Find("SCP"), markerLineWidth);
+        }
     }
+
+    void AddMarker(string name, float RA, float dec, double lst, Color color)
+    {
+        Marker marker = new Marker(name, RA, dec);
+        GameObject markerObject = Instantiate(markerPrefab, this.transform.position, Quaternion.identity);
+        markerObject.transform.parent = allMarkers.transform;
+        MarkerComponent newMarker = markerObject.GetComponent<MarkerComponent>();
+        newMarker.label.text = name;
+        newMarker.markerData = marker;
+        markerObject.name = name;
+        changeStarColor(markerObject, color);
+        if (showHorizonView)
+        {
+            markerObject.transform.position = newMarker.markerData.CalculateHorizonPosition(radius, lst, 0);
+        }
+        else
+        {
+            markerObject.transform.position = newMarker.markerData.CalculateEquitorialPosition(radius);
+        }
+    }
+
+    void AddCircumferenceMarker(string name, Color color, float lineWidth)
```

Comment:
Should rename "name" to "markerName" since the Unity object has a "name" property already

Reply:
Sounds good.

Reply:
Although note in this case I am trying to set the Unity name.

Reply:
This is true, but Visual studio complains (rightly) about the shadowed name. Yes you're changing the name of the object, but it's inside of a monobehaviour script that also has a name property. So unless you're changing the name of the object that this script is running on, this needs to be a different name.

Reply:
Anyway, changed to markerName.